projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -117,13 +130,18 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
       typeof navNode.url === 'string' &&
       this.winRef.nativeWindow?.location.href.includes(navNode.url)
     ) {
-      this.elemRef.nativeElement
-        .querySelectorAll('li.is-open:not(.back), li.is-opened')
-        .forEach((el: any) => {
-          this.renderer.removeClass(el, 'is-open');
-          this.renderer.removeClass(el, 'is-opened');
-        });
-      this.reinitializeMenu();
+      // TODO: (CXSPA-5919) Remove feature flag next major release
+      if (this.featureConfigService.isLevel('6.8')) {
```

Comment:
It looks like earlier versions still need to also call this `reinitializeMenu()` function. So I think we just need to wrap the `this.elementRef.nativeElement...` DOM manipulation with `!6.8`.

Reply:
I've included this invocation in the feature flag.